packages/medusa/src/services/order.ts

```diff
@@ -1393,15 +1440,19 @@ class OrderService extends BaseService {
           break
         }
         case "tax_total": {
-          order.tax_total = await this.totalsService_.getTaxTotal(order)
+          order.tax_total = await this.totalsService_
+            .withTransaction(this.manager_)
```

Comment:
todo: like in the cart service - don't think I would use withTransaction here.

Reply:
Same as the previous one